engine/apps/api/permissions.py

```diff
@@ -265,7 +265,9 @@ def has_object_permission(self, request: AuthenticatedRequest, view: ViewSetOrAP
             for permission_class, actions in rbac_object_permissions.items():
                 if action in actions:
                     return permission_class.has_object_permission(request, view, obj)
-            return False
+
+            # Note: if an endpoint is not found within the rbac_object_permissions dictionary,
+            # that means object permissions are not relevant to this endpoint. Return True (authorized)
```

Comment:
I guess when we've used `rbac_object_permissions` on a `ViewSet` before, we always specify _all_ the endpoints in this object. See [here](https://github.com/grafana/oncall/pull/2597/files#diff-0520dcc9ebfb340cb8390d8fc135fb45364c323abe5eb6bc508e140b58d38ac4R42-R46) for what I'm trying to achieve. tldr; only the beneficiary should be able to PUT/PATCH/DELETE an SSR